_rules/device-motion-user-interface-7677a9.md

```diff
@@ -28,9 +28,17 @@ htmlHintIgnore:
 
 The rule applies to an [HTML document][] with an associated [Window object][] that has an [event listener list][] with one or more [event listeners][event listener] for [device orientation events][device orientation] or [device motion events][device motion].
 
-## Expectation
+## Expectation 1
 
-For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, an [instrument][] is available supporting the same outcomes of the event.
+For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, a set of one or more [instruments][instrument] is available supporting the same outcomes of the event.
+
+## Expectation 2
+
+The [instrument][] is [visible][].
+
+## Expectation 3
+
+The [instrument][] is [included in the accessibility tree][] with an [accessible name][] that is not empty (""). 
```

Comment:
I think that WCAG "user component" does not require a non-empty accessible name…
Unless we consider that "is perceived as…" requires an accessible name for the "as…" part.

Reply:
WCAG's definition of user interface component
> a part of the content that is perceived by users as a single control for a distinct function 

WCAG's example of user interface component
> An applet has a "control" that can be used to move through content by line or page or random access. Since each of these would need to have a name and be settable independently, they would each be a "user interface component." 

From the definition, I would argue that a name is needed for a user to perceive the function. From the example, it's clear it needs to have a name, don't you agree?

Reply:
Fair enough.